components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/gen/java/org/wso2/carbon/identity/configuration/mgt/endpoint/ResourceApiService.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.configuration.mgt.endpoint;
+
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.AttributeDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourceAddDTO;
+
+import javax.ws.rs.core.Response;
+
+public abstract class ResourceApiService {
```

Comment:
add class level and method level comments.

Reply:
Intentionally left due to a generated class.